### PR TITLE
Epilogue: v2.111 added

### PR DESCRIPTION
**Pull request by RosaWagner**
Taken from the upstream repo <https://github.com/Etcetera-Type-Co/Epilogue> at commit <https://github.com/Etcetera-Type-Co/Epilogue/tree/d3aea77a4ac80e394ffabef183d13d36a1c1bc1d>

## Fontbakery report

Fontbakery version: 0.7.27

<details>
<summary><b>[1] Family checks</b></summary>
<details>
<summary>🔥 <b>FAIL:</b> Do we have the latest version of FontBakery installed?</summary>

* [com.google.fonts/check/fontbakery_version](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/universal.html#com.google.fonts/check/fontbakery_version)

* 🔥 **FAIL** Unable to detect what's the latest version of FontBakery available. Maybe we're offline? Please check Internet access and try again.

</details>
<br>
</details>
<details>
<summary><b>[2] Epilogue-Italic[wght].ttf</b></summary>
<details>
<summary>⚠ <b>WARN:</b> METADATA.pb: Fontfamily is listed on Google Fonts API?</summary>

* [com.google.fonts/check/metadata/listed_on_gfonts](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/listed_on_gfonts)

* ⚠ **WARN** Family not found via Google Fonts API. [code: not-found]

</details>
<details>
<summary>⚠ <b>WARN:</b> Is there kerning info for non-ligated sequences?</summary>

* [com.google.fonts/check/kerning_for_non_ligated_sequences](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/kerning_for_non_ligated_sequences)
<pre>--- Rationale ---

Fonts with ligatures should have kerning on the corresponding non-ligated
sequences for text where ligatures aren&#x27;t used (eg
https://github.com/impallari/Raleway/issues/14).

</pre>

* ⚠ **WARN** GPOS table lacks kerning info for the following non-ligated sequences:
	- f + f
	- f + i
	- i + f
	- f + l
	- l + f
	- i + l

   [code: lacks-kern-info]

</details>
<br>
</details>
<details>
<summary><b>[2] Epilogue[wght].ttf</b></summary>
<details>
<summary>⚠ <b>WARN:</b> METADATA.pb: Fontfamily is listed on Google Fonts API?</summary>

* [com.google.fonts/check/metadata/listed_on_gfonts](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/listed_on_gfonts)

* ⚠ **WARN** Family not found via Google Fonts API. [code: not-found]

</details>
<details>
<summary>⚠ <b>WARN:</b> Is there kerning info for non-ligated sequences?</summary>

* [com.google.fonts/check/kerning_for_non_ligated_sequences](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/kerning_for_non_ligated_sequences)
<pre>--- Rationale ---

Fonts with ligatures should have kerning on the corresponding non-ligated
sequences for text where ligatures aren&#x27;t used (eg
https://github.com/impallari/Raleway/issues/14).

</pre>

* ⚠ **WARN** GPOS table lacks kerning info for the following non-ligated sequences:
	- f + f
	- f + i
	- i + f
	- f + l
	- l + f
	- i + l

   [code: lacks-kern-info]

</details>
<br>
</details>

### Summary

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | ℹ INFO | 🍞 PASS | 🔎 DEBUG |
|:-----:|:----:|:----:|:----:|:----:|:----:|:----:|
| 0 | 1 | 4 | 68 | 16 | 247 | 0 |
| 0% | 0% | 1% | 20% | 5% | 74% | 0% |

**Note:** The following loglevels were omitted in this report:
* **SKIP**
* **INFO**
* **PASS**
* **DEBUG**

### CODE REVIEW

File: ofl/epilogue/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<p>
+<em>Epilogue</em> is a sans serif variable font with a weight axis. 9 weights, upright and italic. It supports a wide range of languages in the latin script scope.
```
Comment (m4rc1e) [inline ofl/epilogue/DESCRIPTION.en_us.html:2]: @davelab6 do we support em tags?

Comment (m4rc1e) [reply ofl/epilogue/DESCRIPTION.en_us.html:2]: If you don't know, I can merge once it once I'm happy then check it on dev.

Comment (RosaWagner) [reply ofl/epilogue/DESCRIPTION.en_us.html:2]: Montserrat has an em tag

**Comment by m4rc1e** [comment]
LGTM

**Pull request merged**